cmake/autogenerated_versions.txt

```diff
@@ -1,5 +1,5 @@
 # This strings autochanged from release_lib.sh:
-SET(VERSION_REVISION 54440)
+SET(VERSION_REVISION 54441)
```

Comment:
It's bumped automatically by our release scripts.

Reply:
Yes, but due to https://github.com/ClickHouse/ClickHouse/pull/13156#issuecomment-675633803 perf tests will not work correctly, that's why it has been bumped manually

Reply:
I'm not quite understand what could happen with the performance tests then. There is no changes in the protocol unless the interserver secret is defined. But it's not defined in the performance test. And last time when I changed the protocol I had not to change `cmake/autogenerated_versions.txt` manually.

Reply:
This change looks unnecessary, can you revert it? And then we'll see if the performance tests will fail this time.

Reply:
>And last time when I changed the protocol I had not to change cmake/autogenerated_versions.txt manually.

This depends on what changes you are making

>There is no changes in the protocol unless the interserver secret is defined

I'm afraid this is not correct, since now server \*always\* sends hash for the query (if there is no inter-server secret the hash is empty string).

https://github.com/ClickHouse/ClickHouse/pull/13156/files#diff-43ba0f99a5baf9a6a24378246a48bc06R452

But this can be avoided (in the initial version it was not possible, since it does not have indicator of inter-server not in the query package itself)

Reply:
It's ok because you check `server_revision` before you send the hash (or empty string).

Reply:
And still bumping `VERSION_REVISION` looks strange. This value is changed by our scripts when we release ClickHouse and there is no point to change it more often than the releases of ClickHouse are created. You change `DBMS_TCP_PROTOCOL_VERSION` and that should be enough.

Reply:
upstream/master has 54440 (and this PR will have the same w/o this change), but it does not support cluster-secret, and the client will not be able to communicate with the newer server since it expect the hash for each query (and vice-versa)

Reply:
It's not necessary to support older or newer versions of master, because those versions are not released yet. It's required to make the master compatible only with versions which released or will be released.

Reply:
I suggest finishing this PR using the current versioning system because other versioning system likely should be a subject of another PR.

Reply:
No changes to auto-generated version would be more correct (within the current versioning system).

Reply:
Ok, but note that once this will be merged some PRs may have failed perf tests (the one that do not have this change, but have the same revision)